lib/Installation/Partitioner/LibstorageNG/v4_3/ExpertPartitionerController.pm

```diff
@@ -50,7 +53,14 @@ sub init {
     $self->{AddVolumeGroupPage}            = Installation::Partitioner::LibstorageNG::v4_3::AddVolumeGroupPage->new({app => YuiRestClient::get_app()});
     $self->{AddLogicalVolumePage}          = Installation::Partitioner::LibstorageNG::v4_3::AddLogicalVolumePage->new({app => YuiRestClient::get_app()});
     $self->{ResizePage}                    = Installation::Partitioner::LibstorageNG::v4_3::ResizePage->new({app => YuiRestClient::get_app()});
-
+    $self->{FormattingOptionsPage}         = Installation::Partitioner::LibstorageNG::v4_3::FormattingOptionsPage->new({
+            do_not_format_shortcut  => 'alt-t',
```

Comment:
TBH, I would not mix approaches here. Could you please file follow up ticket to use REST API only?

Reply:
I also planned to add such ticket. I wanted to rewrite the whole FormattingOptionsPage inscope of this PR, but didn't want to make the PR even harder to test and review, as this page is used in running tests already. In the other hand, I didn't want to do double work in terms of adding 'Enable snapshot' method in old approach first and rewrite it again it the following PRs:)

If you think it is critical for us to not mix-up the solutions like that, I can add 'Enable snapshots' in an older way, for sure.

Reply:
It's ok for now, but we don't want to forget about it, so please, file the ticket.